packages/osd-config/src/env.ts

```diff
@@ -36,7 +36,7 @@ import { PackageInfo, EnvironmentMode } from './types';
 export interface EnvOptions {
   configs: string[];
   cliArgs: CliArgs;
-  isDevClusterMaster: boolean;
```

Comment:
Since this is exported, I think the safest option here is to mark this as `@deprecated use isDevClusterManager` and have both options defined.